pyas2lib/as2.py

```diff
@@ -322,6 +322,7 @@ def build(
         content_type="application/edi-consent",
         additional_headers=None,
         disposition_notification_to="no-reply@pyas2.com",
+        domain=None,
```

Comment:
I fell that this needs to be at the Organization level and not at the message level as it would not change per message